<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamsDialog</class>
 <widget class="QWidget" name="ParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>begin vector of compact:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="by_args_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>110</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Solve by args</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>210</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose boarders of compact
(use &quot;,&quot; as separator)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="by_params_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>100</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Solve by params</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>248</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="title_left">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Agruments</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose variables of problem:
(use &quot;,&quot; as separator)</string>
   </property>
  </widget>
  <widget class="QLabel" name="dim_right_title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dimension is:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dim_left_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>170</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dimension is:</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_right">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Parameters</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>20</width>
     <height>331</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="solve_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Solve!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>end vector of compact:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>variables:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>start point:</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>choose epsilon:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="begin_editor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_editor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="start_point_editor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="vector_editor">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="epsilon_editor">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0001</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
